docs/node-operators/get-started/setup-datanode.md

```diff
@@ -326,18 +332,177 @@ vega datanode start --home="YOUR_DATA_NODE_HOME_PATH"
 ## Fetch network history
 After starting a data node, you can load in a segment of network history, if you want your node to have more data than provided by the current height. This is particularly useful if you're running an archive node.
 
+Before you can fetch network history, you will need to update your data node configuration file to add a list of bootstrap peers, as by default this is left empty and depends on which network you are connecting to:
+
+```toml
+[NetworkHistory]
+...
+  [NetworkHistory.Store]
+  ...
+  BootstrapPeers = ["/path/to/bootstrap-peer/1","/path/to/bootstrap-peer/2"]
+```
+
+To get a list of bootstrap peers available for your network, you can make a HTTP request to the API service for your chosen network, for example: `https://api.stagnet1.vega.xyz/api/v2/networkhistory` will retrieve the network peers for `stagnet1`.
```

Comment:
Could we use testnet/fairground here as it's a network someone external could try this with?